examples/simple-form/index.jsx

```diff
@@ -73,13 +73,12 @@ const App = () => (
       <Form>
         <CheckboxField name="bar" label="Do you have pets?" required />
         <TextField name="foo" label="Example" required />
-        {/* <DateField name="baz" required /> */}
         <CheckboxFieldGroup {...checkboxProps} />
+        <FullNameField name='' label=''/>
```

Comment:
If `name` and `label` aren't used in the `FullNameField` component, can they be removed here?